src/pages/Game/Game.js

```diff
@@ -64,6 +78,14 @@ class Game extends Component {
           faceAction={this.setAdvFaceSelect}
           endTurnAction={this.endTurn}
         />
+        <PlayArea
+          adversaryBoard={this.state.game.adversary.board}
+          adversaryDeck={this.state.game.adversary.deck}
+          localBoard={this.state.game.local.board}
+          localDeck={this.state.game.local.deck}
```

Comment:
je pense que tu pourrais deconstruct le state de la meme manière que les props pour éviter de devoir toujours faire `this.state.game.shizzle.othershizzle`